OPHD/UI/StringTable.cpp

```diff
@@ -72,6 +72,26 @@ void StringTable::setVerticalPadding(float padding)
 	mVerticalPadding = padding;
 }
 
+void StringTable::setColumnText(std::size_t column, const std::vector<NAS2D::StringValue>& rows)
+{
+	checkCellIndex({ column, rows.size() - 1});
+
+	for (std::size_t row = 0; row < rows.size(); ++row)
+	{
+		mCells[getCellIndex({ column, row })].text = rows[row].to<std::string>();
```

Comment:
You can potentially shorten the `rows[row].to<std::string>()` to `rows[row].value`.

I guess you could say `StringValue` is basically already a `std::string`, with some conversion functionality added.

Reply:
Ah, thanks. I didn't see the value option for StringValue.